java/maven.embedder/nbproject/project.xml

```diff
@@ -169,6 +169,7 @@
                 <friend>org.netbeans.modules.android.maven</friend>
                 <!-- danis@netbeans.org -->
                 <friend>org.vaadin.netbeans.maven</friend>
+                <friend>org.netbeans.modules.payara.tooling</friend>
```

Comment:
Expanding list of friends, constitutes an API change and has to be properly versioned. See [friend dependencies](http://wiki.apidesign.org/wiki/FriendDependencies) essay.